modules/atomic-widgets/base/atomic-widget-base.php

```diff
@@ -183,15 +183,13 @@ private function transform_setting( $setting ) {
 					: '';
 
 			case 'image':
-				if ( isset( $setting['value']['url'] ) ) {
-					$attachment_url = $setting['value']['url'];
-				}
-
 				if ( isset( $setting['value']['attachmentId'] ) ) {
-					$attachment_url = wp_get_attachment_image_url( $setting['value']['attachmentId'] ) ?? null;
+					$url = wp_get_attachment_image_url( $setting['value']['attachmentId'] ) ?? null;
+				} elseif ( isset( $setting['value']['url'] ) ) {
+					$url = $setting['value']['url'];
 				}
 
-				return $attachment_url ?? Utils::get_placeholder_image_src();
+				return $url ?? Utils::get_placeholder_image_src();
```

Comment:
```suggestion
				return empty( $url ) ? Utils::get_placeholder_image_src() : $url;
```

to catch cases where the URL is falsy but not null